src/OAuth1/AbstractProvider.php

```diff
@@ -135,12 +135,17 @@ public function parseToken($body)
      * @param array $parameters
      * @return \SocialConnect\Common\Http\Response
      */
-    protected function oauthRequest($uri, $method = Client::GET, $parameters = [])
+    public function oauthRequest($uri, $method = Client::GET, $parameters = [])
     {
         $headers = [
             'Accept' => 'application/json'
         ];
 
+        if (isset($parameters['headers']) && is_Array($parameters['headers'])) {
```

Comment:
Cool! Then I'm going to implement that ;)

But I'm only going to do that for the ```OAuth1\AbstractProvider``` I'd say…